components/ConnectButton/ConnectButton.tsx

```diff
@@ -0,0 +1,96 @@
+import { ConnectButton as RainbowConnectButton } from '@rainbow-me/rainbowkit'
+
+const ConnectButton = () => {
+  return (
+    <RainbowConnectButton.Custom>
+      {({
+        account,
+        chain,
+        openAccountModal,
+        openChainModal,
+        openConnectModal,
+        authenticationStatus,
+        mounted,
+      }) => {
+        // Note: If your app doesn't use authentication, you
+        // can remove all 'authenticationStatus' checks
+        const ready = mounted && authenticationStatus !== 'loading'
+        const connected =
+          ready &&
+          account &&
+          chain &&
+          (!authenticationStatus || authenticationStatus === 'authenticated')
+        return (
+          <div
+            {...(!ready && {
+              'aria-hidden': true,
+              style: {
+                opacity: 0,
+                pointerEvents: 'none',
+                userSelect: 'none',
+              },
+            })}
+          >
+            {(() => {
+              if (!connected) {
+                return (
+                  <button
+                    onClick={openConnectModal}
+                    type="button"
+                    className="bg-white min-h-[35px] rounded-full w-[150px] md:w-[225px]"
```

Comment:
lets add 'cursor-pointer' here as well to the connect button when not disconnected.

Reply:
Lol i meant just adding the About Us link to the already existing shown below, and then removing the menu dots whne connected because its not needed anymore.

<img width="594" alt="Screenshot 2023-12-21 at 6 42 41 PM" src="https://github.com/Brick-Creative-Studio/mine-fm/assets/32021205/a27e5342-d1cf-4679-8800-169ae63dc894">